retrieval/target.go

```diff
@@ -30,13 +30,16 @@ import (
 
 	clientmodel "github.com/prometheus/client_golang/model"
 
+	"github.com/prometheus/prometheus/config"
 	"github.com/prometheus/prometheus/storage"
 	"github.com/prometheus/prometheus/utility"
 )
 
 const (
-	// InstanceLabel is the label value used for the instance label.
+	// InstanceLabel is the label name used for the instance label.
 	InstanceLabel clientmodel.LabelName = "instance"
+	// HostLabel is internal label name used for host information.
+	HostLabel clientmodel.LabelName = "__host__"
```

Comment:
Even if it's annoying, these should live here for the moment: https://github.com/prometheus/client_golang/blob/master/model/labelname.go#L20-L44

We can think of moving the whole `model` directory from the client library to the server repo in the future, but for now please open a separate PR for `client_golang` and add cross-referencing comments to each PR.